cmd/tuf/app/key-pop-verify.go

```diff
@@ -18,22 +18,34 @@ package app
 import (
 	"bytes"
 	"context"
+	"crypto"
 	"encoding/base64"
 	"flag"
 	"fmt"
+	"os"
+	"strings"
 
 	"github.com/peterbourgon/ff/v3/ffcli"
+	"github.com/sigstore/root-signing/pkg/keys"
+	"github.com/sigstore/root-signing/pkg/repo"
+	"github.com/sigstore/sigstore/pkg/cryptoutils"
 	"github.com/sigstore/sigstore/pkg/signature"
 	"github.com/sigstore/sigstore/pkg/signature/options"
+	"github.com/theupdateframework/go-tuf"
+	"github.com/theupdateframework/go-tuf/data"
 )
 
+const targetsMeta = "targets.json"
```

Comment:
nit: use `topLevelTargetsFilename`? just because I assume this might be a var name